src/pages/pricing-plans/PricingPlans.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div class="text-textPrimary">
+    <h1 class="text-[32px] md:text-5xl leading-[110%] md:leading-[56px] font-bold">
+      Choose your plan
+    </h1>
+    <div class="py-4 text-lg leading-[26px]">
```

Comment:
> leading-[26px]

Let's try to minimize usage of -[123] tailwind classes.

you can use custom class (as page-title) or just use global default